mcq/mcq.md

```diff
@@ -0,0 +1,151 @@
+# Slide 54
+
+```
+// app.module.ts
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+
+import { AppComponent } from './app.component';
+
+@NgModule({
+  // missing code
+  bootstrap: [AppComponent]
+})
+export class AppModule {}
+```
+
+## What is the missing code?
+
+1. `imports: [BrowserModule, FormsModule], declarations: [AppComponent]`
+
+2. `imports: [AppComponent], declarations: [BrowserModule, FormsModule]`
+
+3. `imports: [NgModule, BrowserModule, FormsModule, AppComponent], exports:
+   [AppModule]`
+
+4. `imports: [BrowserModule, FormsModule], declarations: [AppComponent],
+   exports: [AppComponent]`
+
+# Slide 62
+
+## How to define styles when defining a component?
```

Comment:
Maybe a another way to word this is... "What is one way to define styles when creating a component"
Avoids repetition of the word "define" and doesn't prescribe `styles` as the only true way